src/JSON.jl

```diff
@@ -88,7 +88,7 @@ function _print(io::IO, state::State, s::AbstractString)
     Base.print(io, '"')
 end
 
-function _print(io::IO, state::State, s::Symbol)
+function _print{S <: Union{Symbol,Enum}}(io::IO, state::State, s::S)
```

Comment:
Hi @yuyichao 

Are you suggesting it would be better to just make it

~~~julia
function _print(io::IO, state::State, s)
~~~

? Since submitting this, I now don't think it's a great idea to make a special case for enums so I might just close it. What do you think?